src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1312,7 +1358,7 @@ local function config()
         exitmap[k] = v[1]
         reverse_dirs[v[1]] = v[2]
         short[v[1]] = k
-        local count = #coordmap
+        local count = #coordmap + 1
```

Comment:
I'd probably rename this variable as it does not correspond to the count anymore.